src/overlays/actors/ovl_Demo_Go/z_demo_go.c

```diff
@@ -27,7 +28,7 @@ void func_8097D130(DemoGo* this, GlobalContext* globalCtx);
 void func_8097D290(DemoGo* this, GlobalContext* globalCtx);
 void func_8097D29C(DemoGo* this, GlobalContext* globalCtx);
 
-static u64* D_8097D440[] = { 0x0600CE80, 0x0600D280, 0x0600D680 };
+static u64* sEyeTextures[] = { gGoronCsEyeOpenTex, gGoronCsEyeHalfTex, gGoronCsEyeClosedTex };
```

Comment:
```suggestion
static void* sEyeTextures[] = { gGoronCsEyeOpenTex, gGoronCsEyeHalfTex, gGoronCsEyeClosedTex };
```
We have been using `void*` for these